Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -124,14 +123,20 @@ extension TuistGraph.Target {
         manifest: ProjectDescription.Target,
         generatorPaths: GeneratorPaths
         // swiftlint:disable:next large_tuple
-    ) throws -> (resources: [TuistGraph.ResourceFileElement], playgrounds: [AbsolutePath], invalidResourceGlobs: [InvalidGlob]) {
+    ) throws -> (
+        resources: [TuistGraph.ResourceFileElement],
+        playgrounds: [AbsolutePath],
+        coreDatas: [AbsolutePath],
+        invalidResourceGlobs: [InvalidGlob]
+    ) {
         let resourceFilter = { (path: AbsolutePath) -> Bool in
             TuistGraph.Target.isResource(path: path)
         }
 
         var invalidResourceGlobs: [InvalidGlob] = []
         var resourcesWithoutPlaygrounds: [TuistGraph.ResourceFileElement] = []
         var playgrounds: Set<AbsolutePath> = []
+        var coreDatas: Set<AbsolutePath> = []
```

Comment:
nit: would be `coreDataModels` name be more apt for how it's used?

Reply:
Will do